src/System.Security.Cryptography.X509Certificates/src/System.Security.Cryptography.X509Certificates.csproj

```diff
@@ -16,6 +16,10 @@
     <PackageTargetFramework Condition="'$(PackageTargetFramework)' == ''">dotnet5.4</PackageTargetFramework>
     <UsePackageTargetRuntimeDefaults>true</UsePackageTargetRuntimeDefaults>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetGroup)' == 'netcore50' and '$(ProjectJson)' == '' ">
+    <ProjectJson>netcore50\project.json</ProjectJson>
```

Comment:
The default for this library (for Windows) talks about the Csp library.  I'd expect a package consistency error if the netcore50 tried talking about Csp (which is essentially all the crypto APIs that were disallowed under UWP because they had newer, preferred alternatives).